nautobot/dcim/views.py

```diff
@@ -4450,54 +4450,78 @@ class ControllerUIViewSet(NautobotUIViewSet):
-
-        return context
+    object_detail_content = object_detail.ObjectDetailContent(
+        panels=(
+            object_detail.ObjectFieldsPanel(
+                section=SectionChoices.LEFT_HALF,
+                weight=100,
+                fields=[
+                    "name",
+                    "status",
+                    "role",
+                    "capabilities",
+                    "location",
+                    "platform",
+                    "tenant",
+                    "description",
+                ],
+                value_transforms={
+                    "capabilities": [helpers.label_list],
+                },
+            ),
+            object_detail.ObjectFieldsPanel(
+                section=SectionChoices.RIGHT_HALF,
+                weight=200,
+                label="Integration",
+                fields=[
+                    "external_integration",
+                    "controller_device",
+                    "controller_device_redundancy_group",
+                ],
+            ),
+            object_detail.ObjectsTablePanel(
+                section=SectionChoices.FULL_WIDTH,
+                weight=100,
+                table_class=tables.DeviceTable,
+                table_title="Managed Devices",
+                table_filter="controller_managed_device_group__controller",
+                include_columns=[
+                    "capabilities",
+                    "controller_managed_device_group",
+                    "manufacturer",
+                ],
+                related_field_name="controllers",
+                add_button_route=None,
+            ),
+        ),
+        extra_tabs=(
+            object_detail.DistinctViewTab(
+                weight=700,
+                tab_id="wirelessnetworks",
+                url_name="dcim:controller_wirelessnetworks",
+                label="Wireless Networks",
+                related_object_attribute="device_group_wireless_network_assignments",
+                panels=(
+                    object_detail.ObjectsTablePanelWithPaginator(
+                        section=SectionChoices.FULL_WIDTH,
+                        weight=100,
+                        table_title="Wireless Networks",
+                        table_class=ControllerManagedDeviceGroupWirelessNetworkAssignmentTable,
+                        table_filter="controller_managed_device_group__controller",
+                        tab_id="device_type",
+                        add_button_route=None,
+                        select_related_fields=["wireless_network"],
+                        exclude_columns=["controller"],
+                        header_extra_content_template_path="dcim/wirelessnetworks_header_extra_content.html",
+                    ),
+                ),
+            ),
+        ),
+    )
 
     @action(detail=True, url_path="wireless-networks", url_name="wirelessnetworks", methods=["get"])
     def wirelessnetworks(self, request, *args, **kwargs):
```

Comment:
Can we make sure this view enforces the correct permissions (a la #7369) as part of this PR?